setup.py

```diff
@@ -22,7 +22,8 @@
     cmdclass=versioneer.get_cmdclass(),
     description="Python Image Sequence",
     author="PIMS Contributors",
-    install_requires=['slicerator>=0.9.7', 'six>=1.8', 'numpy>=1.7'],
+    install_requires=['slicerator>=0.9.7', 'six>=1.8', 'numpy>=1.7',
+                      'pillow', 'imageio'],
```

Comment:
:+1: for making imageio a hard dependency. I see that it, in turn, only requires numpy and pillow as required deps: https://github.com/imageio/imageio/blob/master/setup.py#L212